app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -72,6 +67,14 @@ const Contacts = () => {
     );
   };
 
+  useEffect(() => {
+    if (contactIDsToBeDeleted.length === 0) return;
```

Comment:
This is a guard clause. Thus there should be a newline right after this statement. Refer: https://academy.bigbinary.com/learn-react/conditional-rendering/if-else-branching?searchTerms%5B0%5D=Guard&searchTerms%5B1%5D=Clause#p-0-4